.codex/skills/openspec-sync-specs/SKILL.md

```
@@ -1,15 +1,19 @@
-tags: [workflow, specs, experimental]
+name: openspec-sync-specs
+description: Sync delta specs from a change to main specs. Use when the user wants to update main specs with changes from a delta spec, without archiving the change.
+license: MIT
+compatibility: Requires openspec CLI.
+metadata:
+  author: openspec
+  version: "1.0"
+  generatedBy: "1.3.0"
 ---
 
 Sync delta specs from a change to main specs.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)